### DISCUSSION TITLE
Extract posts from a certain time interval

### DISCUSSION THREAD

**Discussion by Ngyx** [kevinzg/facebook-scraper#566] [General]
Hi, 

Thanks so much for this package. I would like to ask if there are options in get_posts to tweak the time period from which the posts gets extracted from a page or group. For example, I'm interested in getting posts from a FB group that were posted in 2015. Is this possible?

**Comment by Ngyx** [comment]
Sorry I think this has been mentioned before and the solution is to pass a start_url argument. I'll try that thanks.

**Reply by neon-ninja** [reply]
Hi, what group are you wanting to do this with? How many posts do they have between now and 2015? Setting `options={"allow_extra_requests": False}` can help speed things along if you're wanting to scrollback quickly.

**Comment by Ngyx** [comment]
Hi, thanks for the reply. I'm doing it for one of my local neighborhood group pages where I live. I managed to extract approximately 500 posts that dated back to 2018 before I was temporarily banned. I was hoping to get posts from 2015.

**Reply by neon-ninja** [reply]
This is possible, but you would need to keep track of the pagination URL. See https://github.com/kevinzg/facebook-scraper/issues/310#issuecomment-852652846
